CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ## [Unreleased]
 None
 
+<a name="4.0.0"></a>
+## [4.0.0] - 2022-02-16
```

Comment:
is the `list_an_application` -> `list_applications` a breaking change?

Reply:
from an SDK standpoint yes.